o-fish-ios/Views/Components/DatePickerWithButton.swift

```diff
@@ -7,16 +7,28 @@
 
 import SwiftUI
 
+enum TypeOfDate {
```

Comment:
move inside the DatePickerWithButton and rename to `Type`

Reply:
Moved to the DatePickerWithButton. Can't rename to Type, because "Type member must not be named 'Type', since it would conflict with the 'foo.Type' expression", so I left the old name "TypeOfDate"

Reply:
make it `PickerType` then